src/client/pages/OfferNew/quote-ids.ts

```diff
@@ -0,0 +1,44 @@
+import queryString from 'querystring'
```

Comment:
Would be awesome if this file was named `useQuoteIds` instead - as all other files exporting hooks are named the same as the hook itself 🙏